finetuner/tuner/paddle/__init__.py

```diff
@@ -221,7 +221,24 @@ def save(self, *args, **kwargs):
         :param args: Arguments to pass to ``paddle.save`` function
         :param kwargs: Keyword arguments to pass to ``paddle.save`` function
         """
-        paddle.save(self.embed_model.state_dict(), *args, **kwargs)
+        if hasattr(self, '_optimizer'):
```

Comment:
In what case would it not have an optimizer?

Reply:
Okay, I think after my PR gets merged you could test normally and we can remove this